.github/workflows/draft-new-release.yaml

```diff
@@ -77,10 +79,9 @@ jobs:
           pr_body: ':crown: *An automated PR*'
 
```

Comment:
To make sure back merge PR (from `main` to `develop`) is the first one to get merged post release, Is it possible to lock `develop` branch as part of this flow ?
Ref: https://github.com/github/lock#setting-a-lock-via-an-issueops-trigger-comment-on-a-pull-request

Reply:
There is no genuine issue with the back merge to develop not being the first PR to merge. However, we should sync the main and develop branches.
I couldn't find a way to programmatically lock branches, as it is part of the branch protection rules.
As this is just a behavioral issue, let's explore other ways to ensure we always carry out this step.

Reply:
Done.